src/runtimes/docker/contracts/ContainerClient.ts

```diff
@@ -609,6 +614,30 @@ type InspectImagesCommand = {
     inspectImages(options: InspectImagesCommandOptions): Promise<PromiseCommandResponse<Array<InspectImagesItem>>>;
 };
 
+export type ImageSbomItem = {
+    /**
+     * The RAW inspect output
+     */
+    raw: string;
+};
+/**
+ * Options for inspecting images
+ */
+export type ImageGenerateSbomCommandOptions = CommonCommandOptions & {
+    /**
+     * The image names/IDs/etc. to inspect, passed directly to the CLI
+     */
+    imageRef: string;
+};
+
+type ImageGenerateSbomCommand = {
+    /**
+     * Generate a CommandResponse for pruning images
+     * @param options Command options
+     */
+    imageGenerateSbom(options: ImageGenerateSbomCommandOptions): Promise<PromiseCommandResponse<Array<ImageSbomItem>>>;
```

Comment:
It doesn't make sense for this to return an array unless the `docker sbom` command can take multiple images; from what I can see, it only accepts a single image tag. This should probably be Promise<PromiseCommandResponse<string>> for the command return type.